src/store/UserStore.ts

```diff
@@ -0,0 +1,21 @@
+import CookieUtils from 'js-cookie';
+
+export class UserStore {
+    // Get the user info via cookie
+    private user = JSON.parse(CookieUtils.get('user') || '{}');    
+    
+    get upn():string {
```

Comment:
why not `get user: User {`
I mean get user object and working with that

Reply:
Good idea